localstack/services/install.py

```diff
@@ -799,6 +799,8 @@ def install(self, package: str, *args, **kwargs):
 
 def main():
     if len(sys.argv) > 1:
+        config.dirs.mkdirs()
+
```

Comment:
Same as above: Is this necessary? It seems to be done implicitly when loading the `config.py` for most cases.